### PR TITLE
[RW-7436][risk=no]Show "Tutorial Workspaces" first in the workspace library

### PR DESCRIPTION
**Pull request by NehaBroad**
https://user-images.githubusercontent.com/34481816/144094859-7cc8caac-25a5-4576-af7f-12fecbe2425c.mov

The assumption is in case Published Workspace is true the order will be  (Pending approval from Shimon)

- Tutorial Workspace( default)
- Demonstration Projects
- Phenotype Library
- Published Workspace

Screen Shot 2021-12-02 at 10 00 16 AM

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Review by aweng98 — APPROVED** [review]

File: ui/src/app/pages/workspace/workspace-library.spec.tsx [hunk 1]
```diff
@@ -85,4 +85,17 @@ describe('WorkspaceLibrary', () => {
     expect(cardNameList).toEqual([publishedWorkspaceStubs[2].name]);
   });
 
+  it('should have tutorial workspaces as default tab', async () => {
```
Comment (jmthibault79) [inline ui/src/app/pages/workspace/workspace-library.spec.tsx:88]: the code change looks good.  I don't really understand this test.

Comment (NehaBroad) [reply ui/src/app/pages/workspace/workspace-library.spec.tsx:88]: Basically if you do not click any tab and launch the component, TUTORIAL workspace is the one that should start showing up.

**Review by jmthibault79 — APPROVED** [review]
approved after updating the tests to make it clear what they're doing by inspection.

Example: 
```
expect(cardNameList).toEqual(TUTORIAL_WORKSPACES)
```
instead of
```
expect(cardNameList).toEqual([publishedWorkspaceStubs[1].name])
```

**Comment by aweng98** [comment]
`cardNameList`: is it a list or single value?!

**Comment by NehaBroad** [comment]
cardNameList is a single card per tab

**Pull request merged**